docs/connecting-data/connecting-to-apis-and-external-data/harmonic-api.md

```diff
@@ -1,9 +1,99 @@
 ---
-description: How to Connect DataDistillr to the Harmonic API
+description: How to Connect DataDistillr to the Harmonic.ai API
 ---
 
-# Connecting to Harmonic
+# Connecting to Harmonic.ai
 
-## First Steps with Harmonic
+## First Steps with Harmonic.ai
+Set up an account with [Harmonic.ai](https://console.harmonic.ai/signup){target=_blank}.
 
-Details coming soon!
\ No newline at end of file
+???+ cost "Cost"
+
+    [Request access](https://www.harmonic.ai/request-access){target=_blank} to find out more about Harmonic.ai. 
+
+
+???+ rlimit "Rate Limits"
+
+    No rate limits on currently supported endpoints
+
+## How to Connect DataDistillr to Harmonic.ai
+To set up a data source connection for Harmonic.ai, you will need to have:
+
+- A unique [name](#name) for your data source connection to be used in queries.
+- The [API Key](#api-key) generated by Harmonic.ai.
+- The [WEBSITE DOMAIN](#website-domain) generated by Harmonic.ai.
```

Comment:
Missing a section for Website Domain

Reply:
I don't see this as something required on the form - if it's only required for querying, add this to the description. Something like 'To set up a data source connection and successfully query Harmonic, you will need to have". Then where you need the Website Domain, explain what it is where to find it and why you need it